src/common/SPELLS_PRIEST.js

```diff
@@ -393,6 +399,13 @@ export default {
     coeff: 0.06,
   },
 
+  HALLOWED_GROUND_TRAIT: {
+    id: 196429,
+    name: 'Hallowed Ground',
+    icon: 'spell_holy_divineprovidence',
+    coeff: 3333,
```

Comment:
I think this value should be in the module, not in SPELLS. Also 'coeff' is a really vague name for this field.

Reply:
I think it should remain here since it's a trait and it's value could be changed in a future patch (unlikely but still possible). Maybe change the variable name but keeping the value itself is good for future-proofing